frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.tsx

```diff
@@ -292,17 +293,13 @@ function StartingJobModal(props: StartingJobModalProps) {
     initialLayerName = fixedSelectedLayer.name;
     initialOutputSegmentationLayerName = getReadableNameOfVolumeLayer(fixedSelectedLayer, tracing);
   }
-  initialOutputSegmentationLayerName = `${initialOutputSegmentationLayerName}_corrected`;
-  // TODO: Other jobs also have an output segmentation layer. The names for them should also be configurable.
-  const hasOutputSegmentationLayer = jobName === "apply merger mode";
+  initialOutputSegmentationLayerName = `${
+    initialOutputSegmentationLayerName || "segmentation"
+  }_corrected`;
+  // TODO: Other jobs also have an output segmentation layer. The names for these jobs should also be configurable.
+  const hasOutputSegmentationLayer = jobName === "materializing this volume annotation";
```

Comment:
This comparison seems rather brittle, I would at least extract the string into a constant and use the constant in both places. Also, I think `materialize volume annotation` would be a more natural job name.

Reply:
I added an enum for the job names that is now used everywhere where jobname strings were used previously

Reply:
Awesome :1st_place_medal: